torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -247,6 +247,73 @@ class FlattenedAssocCommOp : public Expr {
     return sorted_inputs;
   }
 
+  std::vector<EvaluatorValue> evaluate(
```

Comment:
This is not tested right now. But soon in my next PR for adding more simplifications, it will be automatically tested.